### PR TITLE
Changes to Allowance Policy

### PR DESCRIPTION
**Pull request by dmurko**

### CODE REVIEW

File: benefits.md [hunk 1]
```diff
@@ -33,13 +33,17 @@ Conferences count as working days (7.5h per day). Transport to conferences count
...
 
-Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance). In order to not run out, you get a bonus for each conference day:
+Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance). In order to not run out, you get a reimbursal for each away day (conference and travel):
 
 * $50/day for local conferences (in your home town, or close)
-* $100/day for international conferences (you need to fly there, you need a hotel, etc.)
-* $200/day for intercontinental conferences (long flights, extra jetlag days, etc.)
+* $200/day for international conferences (you need to fly there, you need a hotel, etc.)
+* $300/day for intercontinental conferences (long flights, extra jetlag days, etc.)
 * +50%/day if you give a talk on the conference
```
Comment (cas--) [inline benefits.md]: Can we clarify the details of this talk, i.e. length, where and when?

File: benefits.md [hunk 2]
```diff
@@ -33,13 +33,17 @@ Conferences count as working days (7.5h per day). Transport to conferences count
 
 Try to give notice at least 10-14 days before attending a conference.
 
-Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance). In order to not run out, you get a bonus for each conference day:
+Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance). In order to not run out, you get a reimbursal for each away day (conference and travel):
```
Comment (cas--) [inline benefits.md]: Any expense for the conference, such as transport, accommodation, ticket, food, etc. is covered by your [allowance](#productivity-allowance) and the conference allowance which accounts for all the days you are away at the conference, including travelling:

File: benefits.md [hunk 3]
```diff
@@ -33,13 +33,17 @@ Conferences count as working days (7.5h per day). Transport to conferences count
...
-* $100/day for international conferences (you need to fly there, you need a hotel, etc.)
-* $200/day for intercontinental conferences (long flights, extra jetlag days, etc.)
+* $200/day for international conferences (you need to fly there, you need a hotel, etc.)
+* $300/day for intercontinental conferences (long flights, extra jetlag days, etc.)
 * +50%/day if you give a talk on the conference
 
+The conference fee amount is reimbursed to the allowance.
+
+For example: Dejan goes on a three day conference in London. Conference fee is $500, flights are $150, trains and taxis are $150, food is $150, and hotel is $250. Dejan charges the company $1200 as allowance. He had $500 allowance before the trip. He gets $1500 (5 * $200/day + $500 for the conference fee) of conference bonus. Dejan's allowance after the trip is $800.   
+
```
Comment (cas--) [inline benefits.md]:
```
For example, Dejan goes on a three day conference in London:

 * Conference fee is $500. 
 * Flights, trains and taxis are $300.
 * Food and hotel is $400. 

Dejan charges the company $1200 as allowance and his conference allowance is calculated as:

 * $1500 (5 * $200/day + $500 for the conference fee) 

Before the trip Dejan's allowance was $500 in credit and after the trip it is now $800.
```

File: benefits.md (hunk 1 above)
Comment (karantan) [reply benefits.md]: > length

Talks are from 30min up to an hour. You need to coordinate this with conference organisers.

> where and when

Not sure if I understand this one but I would say "on the conference".

Comment (cas--) [reply benefits.md]: I thought it meant a talk about the conference to niteans??

Otherwise it should be 'a talk at the conference'...

Comment (dmurko) [reply benefits.md]: Yep, it needs to say "Talk at the conference.". You need to be a speaker to get the +50% bonus.

**Pull request merged**